core/src/client/client.rs

```diff
@@ -892,7 +892,31 @@ impl BlockProducer for Client {
     }
 }
 
-impl MiningBlockChainClient for Client {}
+impl MiningBlockChainClient for Client {
+    fn get_malicious_users(&self) -> Vec<Address> {
+        self.importer.miner.get_malicious_users()
+    }
+
+    fn release_trusty_users(&self, trusty_vec: Vec<Address>) {
```

Comment:
The objective `trusty user` is not matching with the `malicious user`.
It is actually a method that releases a `malicious user` before.
And it doesn't match with the RPC method name `unban_accounts`

Reply:
I changed the name of `release_trusty_users` to `release_malicious_users`. For internal function names, I used "imprison" and "release" in a pair unlike the rpc name "ban" and "unban". Hence, I kept the verb "release" here. Is it better to unify the rpc name and the name of the internally called function?

Reply:
I think unifying it would be better.